nixos/modules/services/system/uptimed.nix

```diff
@@ -21,12 +21,21 @@ in
   };
 
   config = mkIf cfg.enable {
+
+    environment.systemPackages = [ pkgs.uptimed ];
+
+    systemd.tmpfiles.rules = [
```

Comment:
Isn't this redundant? You now have 3 different processes attempting to create this directory: `createHome = true;`, `tmpfiles`, and `StateDirectory`. Please only use `StateDirectory`.